### PR TITLE
docs(vrl): add docs for new validate flag in punycode functions

### PR DESCRIPTION
**Pull request by esensar**
Related: https://github.com/vectordotdev/vrl/pull/709

### CODE REVIEW

**Review by jszwedko — APPROVED** [review]
This'll need to wait for the VRL release to merge.

File: website/cue/reference/remap/functions/decode_punycode.cue [hunk 1]
```diff
@@ -13,6 +13,13 @@ remap: functions: decode_punycode: {
 			required:    true
 			type: ["string"]
 		},
+		{
+			name:        "validate"
+			description: "Whether to validate the input string to check if it is a valid domain name."
```
Comment (brett0000FF) [inline website/cue/reference/remap/functions/decode_punycode.cue:18]:
```suggestion
			description: "If enabled, checks if the input string it is a valid domain name."
```

File: website/cue/reference/remap/functions/encode_punycode.cue [hunk 2]
```diff
@@ -13,6 +13,13 @@ remap: functions: encode_punycode: {
 			required:    true
 			type: ["string"]
 		},
+		{
+			name:        "validate"
+			description: "Whether to validate the input string to check if it is a valid domain name."
```
Comment (brett0000FF) [inline website/cue/reference/remap/functions/encode_punycode.cue:18]:
```suggestion
			description: "If enabled, checks if the input string it is a valid domain name."
```

**Review by brett0000FF — APPROVED** [review]
Looks good. Just a small suggestion for the description text.

**Review by pront — APPROVED** [review]
Thanks @esensar!

**Pull request merged**